index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <div class="header__bg">
+        <img src="img/bg.png" alt="bg" class="header__bg-content" />
+      </div>
+      <div class="header__flex-container">
+        <div class="header__flex-container-logobox">My Logo</div>
+        <div class="header__flex-container-menu">
+          <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </div>
```

Comment:
- [ ] Lets keep html semantic and use `nav` tag. I suggest to wrap the navbar with` <nav>` tag